Scripts/run-linux-tests.sh

```diff
@@ -0,0 +1 @@
+  
```

Comment:
Not sure what this file is for.

Reply:
I'll remove that

Reply:
It might be worthwhile to keep it and have it run the tests in a Linux environment -- in the same way that the `run-ios-tests.sh` does in a MacOS environment.

Reply:
Yes I agree. Also let me explore our options with docker some more. If we can have Travis spin up the Linux tests container using the swift image from the start, then we can have all the benefits without doing the nested container thing

Reply:
So I haven't found much in the way of avoiding the nested container. We either have a choice of building our Swift environment in Linux directly in the travis container, or building a docker image based on the `swiftdocker` image to run the tests in a nested container.

Which direction do we want to go? I personally vote for using the docker image, for the reason that we get a clean environment to run the swift linux tests in without having to worry about pulling down linux images, pgp verification, etc.

Reply:
I suppose it is better to use a prepackaged image that to create one ourselves, so let's go with the Docker image.